src/ucp/core/ucp_request.h

```diff
@@ -349,6 +375,10 @@ struct ucp_request {
             ucp_ep_ext_gen_t        *next_ep;   /* Next endpoint to flush */
         } flush_worker;
     };
+
+    #if ENABLE_DEBUG_DATA
```

Comment:
remove spaces before `#`, i.e. start from the beginning of the line
pls, apply for all places